includes/admin/class-omise-page-card-form-customization.php

```diff
@@ -0,0 +1,130 @@
+<?php
+
+defined('ABSPATH') or die('No direct script access allowed.');
+
+if (class_exists('Omise_Page_Settings')) {
+	return;
+}
+
+class Omise_Page_Card_From_Customization extends Omise_Admin_Page
+{
+
+	const PAGE_NAME = 'omise_card_form_customization_option';
+
+	protected function get_default_design_setting()
+	{
+		$theme = (new Omise_Payment_Creditcard())->get_option('card_form_theme');;
+		if(empty($theme)) {
+			$theme = 'light';
+		}
+		$design = [
+			'light' => [
+				'font' => [
+					'name' => 'Poppins',
+					'size' => 16,
+				],
+				'input' => [
+					'height' => '44px',
+					'border_radius' => '4px',
+					'border_color' => '#ced3de',
+					'active_border_color' => '#1451cc',
+					'background_color' => '#ffffff',
+					'label_color' => '#212121',
+					'text_color' => '#212121',
+					'placeholder_color' => '#98a1b2',
+				],
+				'checkbox' => [
+					'text_color' => '#1c2433',
+					'theme_color' => '#1451cc',
+				]
+			],
+			'dark' => [
+				'font' => [
+					'name' => 'Poppins',
+					'size' => 16,
+				],
+				'input' => [
+					'height' => '44px',
+					'border_radius' => '4px',
+					'border_color' => '#475266',
+					'active_border_color' => '#475266',
+					'background_color' => '#131926',
+					'label_color' => '#E6EAF2',
+					'text_color' => '#212121',
+					'placeholder_color' => '#DBDBDB',
+				],
+				'checkbox' => [
+					'text_color' => '#E6EAF2',
+					'theme_color' => '#1451CC',
+				]
+			]
+		];
+		return $design[$theme] ?? $design['light'];
```

Comment:
This will break for everyone using below PHP 7. Let's use normal ternary operator until we decide to not support PHP 5.